doc/manual.rst

```diff
@@ -5118,6 +5118,50 @@ scope is the default.
 ``bind`` statements only make sense in templates and generics.
 
 
+Delegating mixin statements
```

Comment:
* this example has many bugs (symbols should be exported, `var O` instead of `O`, etc
* after fixing those locally, the example works both before and after PR, so it's probably not the best example